group_vars/all.sample

```diff
@@ -186,6 +186,11 @@ dummy:
 # Object backend
 #osd_objectstore: filestore
 
+# xattrs. by default, 'filestore xattr use omap' is set to 'true' if
+# ext4 if 'osd_mkfs_type' is set to 'ext4'; otherwise it isn't set. This
+# can be set to 'true' or 'false' to explicitly override those defaults.
+# filestore_xattr_use_omap: null
```

Comment:
Docs fixed. I think I got lost halfway through that sentence.

`false`/`null` doesn't really matter; it just needs to be a non-true value. But I think `null` is more intuitive, since the difference between `false` (the boolean) and `"false"` (the string) is super critical here and not super obvious. But if you've got religion about this being `false` I'll change it.